playbook/doom-dev/main.tf

```diff
@@ -52,6 +52,26 @@ variable "hydra_node_image" {
   description = "The Docker image to use for the Hydra node component."
 }
 
+variable "sidecar_image" {
+  type        = string
+  description = "The Docker image to use for the sidecar component of the Hydra node."
+}
+
+variable "dedicated_image" {
+  type        = string
+  description = "The Docker image to use for the dedicated / referee server of the hydra node."
+}
+
+variable "open_head_image" {
+  type        = string
+  description = "The Docker image to use for the open head component of the Hydra node."
+}
+
+variable "control_plane_image" {
+  type        = string
+  description = "The Docker image to use for the control plane component of the Hydra node."
+}
+
```

Comment:
You should only need the `dedicated_image` var. The other should all match the main image that we build out of this same repo.

Reply:
oh i think this might be a merge snafu